### PR TITLE
Improve typography units

### LINKED ISSUE #208: Typography: use of % units.
- [x] Letter-spacing: REMOVE - its not properly supported
- [x] Margin bottom: REMOVE - creates a variable result - the margin is calculated on the text elements container width. not relative to the font-size like EM is.
- [x] Line-height: If it stays then change slider range to be INT and increase range eg. 0 to 250.
- [x] Font-size: increase range.

Overall not sure the value of % but some users ask for it.

### PR DESCRIPTION
**Pull request by tomusborne**
Closes #208 

This PR does the following:

- Removes the percentage unit from properties it doesn't work with
- Removes the default placeholders if the unit doesn't match the default
- Adjusts the range control min/max if using percentages
- Adds `rem` as a unit
- Adds an HTML element you can choose (to set the base font size for `rem`)
- Sets the unit on element change if a default unit exists
- Uses the `value` prop instead of `placeholder` for the defaults

### CODE REVIEW

File: src/customizer-controls/font-manager/placeholders.js [hunk 1]
```diff
@@ -3,6 +3,20 @@ import {
 } from '@wordpress/i18n';
 
 const elements = {
+	html: {
+		module: 'core',
+		group: 'base',
+		label: 'html',
```
Comment (JeanPaiva) [inline src/customizer-controls/font-manager/placeholders.js:9]: Myabe "HTML" all uppercase? It's kind weird to see it all lowercase.

**Review by JeanPaiva — COMMENTED** [review]
I think is better to add the value instead of the placeholder. Will avoid the user seeing "17" and trying to increase to 18 but instead gets a 1 value.

**Comment by tomusborne** [comment]
Was playing with that and trying to find if it's possible to start the arrows at the placeholder - couldn't find anything.

I'll replace with the value in this PR.

**Review by JeanPaiva — APPROVED** [review]
All good now

**Pull request merged**